packages/ckeditor5-easy-image/src/cloudservicesuploadadapter.ts

```diff
@@ -50,10 +51,10 @@ export default class CloudServicesUploadAdapter extends Plugin {
 			return;
 		}
 
-		this._uploadGateway = editor.plugins.get( 'CloudServicesCore' ).createUploadGateway( token, uploadUrl );
+		this._uploadGateway = ( editor.plugins.get( 'CloudServicesCore' ) as CloudServicesCore ).createUploadGateway( token, uploadUrl! );
```

Comment:
For readibility I would suggest:

```suggestion
		const cloudServicesCore = editor.plugins.get( 'CloudServicesCore' ) as CloudServicesCore;

		this._uploadGateway = cloudServicesCore.createUploadGateway( token, uploadUrl! );
```